src/index.js

```diff
@@ -11,4 +11,38 @@ function scoreElement(name, score) {
   tableRow.appendChild(tableData);
   tableBody.appendChild(tableRow);
 }
-scores.forEach((score) => scoreElement(score.user, score.score));
+async function getScores() {
+  // Get scores data
+
+  const data = await fetch(requestURL);
+
+  // convert scores data to object/array
+  const response = await data.json();
+
+  // find scores array
+  const scores = await response.result;
+  return scores;
+}
+async function addScores(user, score) {
+  fetch(requestURL, {
```

Comment:
- [ ] Kindly make use of arrow functions instead of the function keyword to follow ES6 best practices.